path.go

```diff
@@ -0,0 +1,56 @@
+package txtdirect
+
+import (
+	"fmt"
+	"net"
+	"regexp"
+	"strings"
+)
+
+func zoneFromPath(host string, path string) (string, int, error) {
+	if path == "/" {
```

Comment:
What's the reasoning behind this check here?

Reply:
I did a double check and realized it's useless because at first we will get host's TXT record before calling this function, so there is no need to return host's zone in this function. Will remove it.